### ISSUE TITLE
fluent-sqlserver

### ISSUE THREAD

**Issue by richgabrielli** [vapor/fluent-kit#297]
Any thoughts / ambitions on creating connectivity to Microsoft SQL Server?

**Comment by thejdah** [comment]
Any update on this?

**Comment by 0xTim** [comment]
@thejdah Nope, essentially waiting for someone in the community to build a driver. It's not on the Vapor roadmap
